README.md

```diff
@@ -19,7 +19,7 @@ This is a proposed template repository for applications for future developments.
 1. npm package manager.
 1. Gatsby command line interface (CLI). 
 1. A GitHub account.
-1. A Microsoft Azure account (to deploy with Azure).
+1. A Microsoft Azure account (if deploying with Azure). 
 
 ## Getting Started
 1. Click the `Use this template` button.
```

Comment:
This [repo](https://github.com/18F/federalist-uswds-gatsby)'s documentation has a section that has these steps. The documentation, in general, can be helpful.